Kiosk/App/Views/RegisterFlowView.swift

```diff
@@ -111,3 +121,11 @@ class RegisterFlowView: ORStackView {
         }
     }
 }
+
+extension Array {
```

Comment:
Extensions are scoped as normal types are, either public, private, or internal. The default is internal, which means scoped to the target (the Kiosk app target, in this case).